src/components/Plugin/SafeSnap/CustomBlock.vue

```diff
@@ -0,0 +1,307 @@
+<template>
+  <Block
+    title="SafeSnap"
+    :icon="!loading && moduleAddress ? 'refresh' : undefined"
+    @submit="updateDetails"
+  >
+    <div class="mb-3 text-center">
```

Comment:
This block seem to appear when `infoLabel` is empty, creating some extra margin 
![image](https://user-images.githubusercontent.com/16245250/119710214-4eb13c80-be88-11eb-94f0-d852227927bb.png)
It should be better to do like this:
```suggestion
    <div v-if="infoLabel" class="mb-3 text-center">
```